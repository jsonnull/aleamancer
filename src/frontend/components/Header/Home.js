// @flow
import React from 'react'
import { Link } from 'react-router-dom'
import Tooltip from '../Tooltip'
import Button from './Button'

const Home = () => (
  <Tooltip placement="right" content="Home">
    <Link to="/sessions">
      <Button square>
        <i className="fa fa-home" />
      </Button>
    </Link>
  </Tooltip>
)

export default Home
