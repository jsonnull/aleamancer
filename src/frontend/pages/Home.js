// @flow
import React from 'react'
import styled from 'styled-components'
import { light as theme } from '../styles/themes'
import { colors } from 'frontend/styles/common'
import Logo from 'frontend/components/Logo'

const Container = styled.div`
  position: absolute;
  top: 0;
  bottom: 0;
  right: 0;
  left: 0;
  background-color: ${colors.blue};
  color: ${theme.background};
  display: flex;
`

const Item = styled.div`
  margin: auto;
  text-align: center;
`

const Home = () => {
  return (
    <Container>
      <Item>
        <Logo height={'40px'} />
      </Item>
    </Container>
  )
}

export default Home
