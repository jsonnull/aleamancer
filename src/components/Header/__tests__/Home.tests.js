// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../Home.js'

describe('Header Home component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Home goHome={() => {}} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
