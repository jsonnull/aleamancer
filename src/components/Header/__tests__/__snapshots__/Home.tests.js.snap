// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Home component renders correctly 1`] = `
<div
  className="sc-bdVaJa kgrFwP"
  onClick={[Function]}
>
  <i
    className="fa fa-home"
  />
</div>
`;
